plugins/modules/zos_lineinfile.py

```diff
@@ -361,6 +367,7 @@ def main():
         backup_name=dict(type='str', required=False, default=None),
         firstmatch=dict(type='bool', default=False),
         encoding=dict(type='str', default="IBM-1047"),
+        tmphlq=dict(type='str', required=False, default=None)
```

Comment:
There is confusion... what I thought I saw was 'tmphlq' is a string/parameter, and tmp_hlq is a dictionary. As I look more closely, I'm seeing them intermixed. I'll try to put comment notes where I see tmphlq as a dictionary.

Reply:
Resolving because I created issue https://github.com/ansible-collections/ibm_zos_core/issues/387 - it was a good comment and point, please review the issue as well @fernandofloresg , also see the other work item that came from this PRs review https://github.com/ansible-collections/ibm_zos_core/issues/386